README.md

```diff
@@ -116,34 +116,113 @@ This allow to **configure** the layouts **on the go** while using it.
 
 When opening a **new session** Material Shell will **restore every windows** previously present with a "**window placeholder** which allow you to reopen any of the previous window easily at the same spot you like to have them.
 
+
+## Brief Tutorial
+Getting around using the mouse and keyboard shortcuts
+
+Setup
+
+1. The first workspace will be used to view and refer to this README.md file. The 2nd, 3rd,.. 10th will be used for Material-Shell practice.
+2. In the first workspace, click on the tiling icon in the upper right corner. Click on `Tweak available layout`.
+3. Turn off all layout styles except `Float`. 
+4. Remember that you can return to workspace one, this README file, by using `Super+1`. 
+5. To open a second workspace, click on the `+` on the left side of the screen, or use or `Super+0`.
+6. Turn on the first 6 tiling choices on, e.g., `Maximize, Split ... Half vertical`. Turn off the remaining layouts, especially `Float`. Click on `Confirm layouts` to save.
+7. Click on the tiling icon again and select `Split` from the list of layouts. Increase the number of layouts to `4`.
+8. Use `Super+x` to open App Launcher, and then open a terminal window. Do this 4 times. There should 4 (Split) windows on the desktop.
+9. Execute a command in each terminal instance so that the terminal windows are distinguishable from each other. For example, execute `ls, nano, man ls, htop` respectively.
+10. Use `Super+0` to open another workspace, and open two applications, for example, an email client and Libre Writer.
+11. Use `Super+End` to open another workspace, and  open an image editor.
+
+Practice
+
+1. Cycle through the tiling layouts using `Super+Spacebar`. Notice that the tiling-layout icon updates to represent the current layout.
+2. Experiment with keyboard shortcuts. Try the following:
+	-Move the focus within a workspace, to another workspace, and if applicable, to another monitor.
+	-Move a window within a workspace, to another workspace, and if applicable, to another monitor.
+	-Resize a window.
+	-Close an application.
+
+Take note of the following:
+
+1. The workspace icons are located on the left side of the screen and the window panes (tabs) are located at the top of the screen.
+2. When using a browser, the browser tabs are located below the window panes. Browser tabs and window panes can be similar in appearance.
+3. There isn't a keyboard/mouse equivalent of `Maximize` window. Use the tiling layout icon or `Super+spacebar` to return to a maximized window.
+4. Tiling layout preferences are unique to the each workspace (the preferences do not behave as workspace global settings).
+5. Note that a few Material-Shell keybindings duplicate some Gnome keybindings e.g. `Super+s, Super+PgDown`, `Super+Home`, `Super+End` etc..
```

Comment:
This is a nice addition but I would prefer move it to an external readme file `documentation/tutorial.md` and leave a link to it in this file.

Reply:
> Thanks you for choosing Material Shell for your first contribution !
> 
> In theses pull request reviews I mark some annotions for changes I would like to happen before merging your work in the Material Shell project.
> 
> For your next contribution I would suggest you to make a proposal of what you will do as contribution because even if projects are open sources not every contributions can be merged. To prevent spending lot of time working on something that can't be merged for various reason can be extremely frustrating.
> 
> It's safer to start by making a proposal so the existing contributors can discuss and debate before the work is even started to be sure the work is going in the right direction.
> 
> For exemple in our case we agreed on updating the hotkeys documentation but I wasn't aware that you would work on creating a Tutorial for Material Shell.
> 
> In our case it's not a problem since it's a nice addition I will merge it but some time it's not possible.
> 
> You can see that there is old pull request that has never been merged. Most of them are related to independent initiative that cannot be merged and was a waste of time.
> 
> Anyway thanks for your work !

You're welcome, my pleasure. Thanks to pandemic life I have a lot of time on my hands and looking for something to do besides job searching and tutorials. Also, I want a  predictable desktop, where, using hotkeys and muscle memory, learn to fly through my various windows and applications. My favorite layout combination so far is the Max and Half. The Half mimics a Split with two windows/apps, and can handles pop-ups on Fedora with ease. Max is, well max. Looking forward to working on it some more. Thanks!